lfs/client.go

```diff
@@ -200,16 +200,21 @@ func Batch(objects []*objectResource, operation string) ([]*objectResource, *Wra
 	tracerx.Printf("api: batch %d files", len(objects))
 	res, objs, wErr := doApiBatchRequest(req, creds)
 	if wErr != nil {
-		if res != nil {
-			switch res.StatusCode {
-			case 404, 410:
-				tracerx.Printf("api: batch not implemented: %d", res.StatusCode)
-				sendApiEvent(apiEventFail)
-				return nil, Error(newNotImplError())
-			}
+		if res == nil {
+			sendApiEvent(apiEventFail)
+			return nil, wErr
+		}
+
+		switch res.StatusCode {
+		case 401:
+			Config.SetPrivateAccess()
+			tracerx.Printf("api: batch not authorized, submitting with auth")
```

Comment:
I might be leaning toward putting a message in both. My thinking is: putting one at this level indicates that the auth failed and it's resubmitting, this is the appropriate level for that message. Having one in `Config.SetPrivateAccess()` shows the config setting being changed, which could come from anywhere (even though this is currently the only place it is called).